workbench/tests/main/main.test.js

```diff
@@ -225,7 +225,8 @@ describe('createWindow', () => {
 });
 
 describe('investUsageLogger', () => {
-  const expectedURL = `http://127.0.0.1:${process.env.PORT}/api/log_model_start`;
+  const PORT = 3000;
```

Comment:
This constant could be anything, right? We're mocking the actual fetch and only confirming that the URL is being constructed in the intended way. Should we add a comment here so it's clear we're not actually using a port in this test?

Reply:
Added a comment in https://github.com/natcap/invest/pull/1691/commits/8331349f42fa6799ac42cb81ff165e42285542cb